### ISSUE TITLE
Support Multiread & Multiwrite on sd_card.sv

### ISSUE THREAD

**Issue by naeloob** [MiSTer-devel/Template_MiSTer#7]
Hi, my question is if it's supported multiwrites and reads on the sd_card.sv wrapper.

Else, could be implemented?

Thanks in advance.

Regards.

**Comment by sorgelig** [comment]
it's supported, but module itself is not very robust and may not work with some cores. If it doesn't work then it needs to be fixed if there is such core.
Better, if possible use direct MiSTer API.

**Comment by naeloob** [comment]
Well, i'm tring to port a core and i'm having problems with the sd card from inside the computer itself, but the vhd works in the core, i mean it loads rom and configs from the core, but the sd driver inside the computer cannot work with the sd.

First try to load information about the real sd card from CSD, OCR and CID registers, and it seems that sd_card doesn't provide it. I have hardcoded this information on the drive... And that driver works with multiread and multiwrite commands.

It still doesn't works, i dont know if its caused by my hardcoded parameters, by the multi* or something that i missed...

It seems that Multi* was solved on mist, i will try to port it to Mister and see if i can make some steps fordward. I have work ahead :)

Thanks!!
